bokeh/models/annotations.py

```diff
@@ -220,3 +222,21 @@ class Span(Annotation):
     line_props = Include(LineProps, use_prefix=False, help="""
     The %s values for the span.
     """)
+
+class Tooltip(Annotation):
+    """ Render a tooltip.
+
+    .. note::
+        This model is typically managed by BokehJS and is not useful
+        directly from python.
+
```

Comment:
I mean we don't actually (currently) ever create or set a tooltip from python, but the class needs to exist to please our machinery. I'm not sure we will either, on the JS side it creates a tooltip per renderer. I think we need a concept of "client originated" models .